eng/native/configureplatform.cmake

```diff
@@ -275,7 +275,7 @@ if (CLR_CMAKE_TARGET_OS STREQUAL CLR_CMAKE_HOST_OS)
         endif()
     endif()
 else()
-    if(NOT (CLR_CMAKE_HOST_OS STREQUAL Windows_NT))
+    if(NOT (CLR_CMAKE_HOST_OS STREQUAL win))
```

Comment:
All the other `CLR_CMAKE_HOST_OS` values on other platform are initialized to longer names that start with upper case so this makes it inconsistent.

You could change it to `Windows` on L117 instead.